.bazelrc

```diff
@@ -72,6 +72,19 @@ build:cuda --@xla//xla/python:enable_gpu=true
 build:cuda --@xla//xla/python:jax_cuda_pip_rpaths=true
 build:cuda --define=xla_python_enable_gpu=true
 
```

Comment:
Please don't add the extra newline

Reply:
I disagree. It's much more readable with it.

Reply:
I disagree with you. Usually you follow the style used throughout the file.

All configs are clubbed together to avoid confusion which is how I structured it.

But I guess there is no readability or formatter here so we'll both just do whatever we want anyways

Reply:
It's much clearer to my eyes if the large comment is grouped only with the lines to which it applies.